src/Internals/WCCom/WCConnectionHelper.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Helpers for managing connection to WooCommerce.com.
+ *
+ * @package Automattic\WooCommerce\Internals\WCCom
+ */
+
+namespace Automattic\WooCommerce\Internals\WCCom;
```

Comment:
Just noting that the namespace should be `Automattic\WooCommerce\Internal\WCCom`!

Reply:
Good catch! Changed in https://github.com/woocommerce/woocommerce/pull/26932/commits/4500e86198729ce893945a39fe9e22cee3991856